File: Funcional/ATV_1_LISTA_1/exercicio12.py
def calculoLucro(valor):
    if valor < 30.00:
        return("O valor de venda sera R${:.2f}".format(valor + (valor * 0.45)))
    return("O valor de venda sera R${:.2f}".format(valor + (valor * 0.30)))

print(calculoLucro(30.00))

File: Funcional/ATV_1_LISTA_1/src/exercicio12.hs
calculoLucro valor
    | valor < 30.00 = "O valor de venda sera R$" ++ show(valor + (valor * 0.45))
    | otherwise = "O valor de venda sera R$" ++ show(valor + (valor * 0.30))
    
main = do
    print (calculoLucro 20.00)
    print (calculoLucro 30.00)
    print (calculoLucro 40.00)